bcipy/simulator/tests/test_button_press_model.py

```diff
@@ -0,0 +1,71 @@
+"""Tests for Button Press Model"""
```

Comment:
We can do this in another PR, but there are enough tests for this module now that we should follow the pattern set elsewhere.

`tests/*submodule_name*/test_*name_of_file_tested*.py`